test_examples/module_emr-iam/README.md

```diff
@@ -0,0 +1,3 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Is this to pass make lint?

Reply:
If I had to guess it was `make terraform/docs`, but @hbarros-caylent can confirm